src/kaas_jobs.ts

```diff
@@ -76,8 +94,41 @@ export async function pollForJobStatus(
         );
         testRun.passed(test, jobDetails.duration * 1000);
         testRun.end();
+
+        // Automatically display the report
+        // Kontrol job
+        if (jobDetails.kind === JobKind.kontrol) {
+          if (jobDetails.children && jobDetails.children.length > 0) {
+            for (const childJob of jobDetails.children) {
+              const report = await getJobReportByJobId(client, childJob.id);
+              if (report) {
+                const reportUrl = jobReportUri(childJob).toString();
+                createAuthenticatedWebview(
+                  reportUrl,
+                  `jobReport-${childJob.id}`,
+                  `Job ${childJob.id.slice(0, 6)} Report`
```

Comment:
Does opening the report automatically for potentially long-running compute jobs make sense? From a user perspective, it might be confusing when the report opens multiple hours after the job was started.